engines/scumm/saveload.cpp

```diff
@@ -837,6 +837,7 @@ static void syncWithSerializer(Common::Serializer &s, VerbSlot &vs) {
 	s.syncAsByte(vs.center, VER(8));
 	s.syncAsByte(vs.prep, VER(8));
 	s.syncAsUint16LE(vs.imgindex, VER(8));
+	vs.origLeft = -1;
```

Comment:
Ahh now I see that if I save the game and load it, left and origLeft get wrong values, and the text becomes left-aligned :/